src/main/java/com/zufar/onlinestore/cart/endpoint/CartEndpoint.java

```diff
@@ -30,14 +31,14 @@
 @RequiredArgsConstructor
 @Validated
 @RequestMapping(value = CartEndpoint.CART_URL)
-public class CartEndpoint {
+public class CartEndpoint implements CartsApi {
```

Comment:
CartApi sounds better

Reply:
We are already have CartApi in this class (import com.zufar.onlinestore.cart.api.CartApi). Is it ok to use same names?

Reply:
Yes, there is no problem